test/ttmlir/Silicon/TTMetal/simple_constant.mlir

```diff
@@ -1,3 +1,7 @@
+// REQUIRES: wormhole_b0
+// REQUIRES: functional,perf
+// REQUIRES: n150,n300
+// REQUIRES: push
```

Comment:
If all of these REQUIRES are just for testing this PR that's OK.  Otherwise I don't think they should be listed.

Reply:
yeah its just for testing